test/JDBC/expected/BABEL-949.out

```diff
@@ -0,0 +1,36 @@
+CREATE SEQUENCE seq_949 AS INT START WITH 1 INCREMENT BY 1;
```

Comment:
1. Let's try to use more descriptive test file name, e.g., `BABEL-NEXT-VALUE-FOR`.
2. Please make this as an upgrade test and put it in the latest and 14_7 schedule files.
3. The prepare script would be exactly same as this file and in the verification script, you need to further check INSERT and SELECT NEXT VALUE.